test/cpp/qps/qps_driver.cc

```diff
@@ -68,49 +82,124 @@ using grpc::testing::ServerType;
 using grpc::testing::RpcType;
 using grpc::testing::ResourceUsage;
 
+using std::vector;
+using std::string;
+
+vector<string> ParseCommaSeparatedIntList(string list) {
+  // Pass by value because strtok consumes its input.
+  vector<string> result;
+  char *p = strtok((char*)list.c_str(), ",");
+  while (p) {
+    result.push_back(p);
+    p = strtok(NULL, ",");
+  }
+  return result;
+}
+
+vector<ClientConfig> GetClientConfigsFromFlags() {
+  vector<ClientConfig> configs;
+  const auto rpc_types = ParseCommaSeparatedIntList(FLAGS_rpc_types);
+  const auto outstanding_rpcs_per_channel =
+      ParseCommaSeparatedIntList(FLAGS_outstanding_rpcs_per_channel);
+  const auto client_channels =
+      ParseCommaSeparatedIntList(FLAGS_client_channels);
+  const auto payload_sizes = ParseCommaSeparatedIntList(FLAGS_payload_sizes);
+  const auto client_types = ParseCommaSeparatedIntList(FLAGS_client_types);
+  const auto async_client_threads =
+      ParseCommaSeparatedIntList(FLAGS_async_client_threads);
+
+  ClientConfig cc;
+  cc.set_enable_ssl(FLAGS_enable_ssl);
+
+  for (const auto& rpc_type_str : rpc_types) {
```

Comment:
I'll address the other question later, but for now let me say this. I would oppose re-introducing range-based for in to any grpc code, including the qps code. I realize that qps is currently still not compatible with VS2010 or gcc4.4 because of lambda captures and local templates, but we've already removed range-based for and I wouldn't want to remove it again.